free-1.md

```diff
@@ -1,68 +1,170 @@
 # Free project 1
 
+
+
+An RPG mapmaker with mixed tile sizes and shapes. It would let users create or upload tiles of certain sizes, then specify on the map which patterns belong where by clicking and dragging map regions and selecing the tile to fill the area. There would also be an option to designate what a "border" to the tile looks like, and then the area would have the border on the outside. Tiles that don't fill a square could also have transparency and be layered over others, so borders wouldn't have to be sharp and linear. Once the areas and patterns are specified, an image would be produced. "Objects" could also be placed on the scene, layered above tiled areas.
+
+
+
 ## The user and a language
+
 This section describes who the project would serve and why a language might be a
+
 good way to meet their needs.
 
 
+
+The user would be game developers (or perhaps bored artists). Unfortunately, this feels more like an application than a language, but map generation is a very useful tool.
```

Comment:
I agree that this may end up being more like designing some sort of UI.
That being said, is a UI a DSL? Doesn't seem like too much of a stretch to me.